bio/hisat2/index/wrapper.py

```
@@ -22,14 +22,7 @@
     input_seq += "-c "
 input_seq += ",".join(fasta) if isinstance(fasta, list) else fasta
 
-hisat_dir = snakemake.params.get("prefix", "")
-if hisat_dir:
-    os.makedirs(hisat_dir)
+# get common prefix
+prefix = os.path.commonprefix(snakemake.output).rstrip(".")
```

Ruff

Undefined name `snakemake`

(F821)